templates/init.go

```diff
@@ -24,33 +27,35 @@ const (
 
 //go:generate go run generate/generate.go
 func init() {
-	err := fs.WalkDir(yamlTemplates, ".", func(filepath string, d fs.DirEntry, err error) error {
-		if err != nil {
-			return err
-		}
-		if d.IsDir() {
-			return nil
-		}
+	templatesOnce.Do(func() {
```

Comment:
äh.... du machst das 1x, aber immer noch im init, also IMMER. So bringt das nix ;)

Reply:
Ja. Ändere es nachher. War in Eile.

Reply:
Habs nochmal auf eine andere Art gemacht.